### PR TITLE
BUG005 fix image polymorphic relation

### PR DESCRIPTION
**Pull request by YanPam**
Fix Image polymorphic relation
[card](https://trello.com/c/papevg1r/56-bug005-fix-image-polymorphic-relatoin)

### CODE REVIEW

File: app/models/musician_skill.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class MusicianSkill < ApplicationRecord
+  has_many :musician_skill_user
+  has_many :users, through: :musician_skill_user
```
Comment (MariaPaharelava) [inline app/models/musician_skill.rb]: has_many :musician_skills, through: :musician_skill_users ???

File: db/migrate/20190808073147_create_musician_skill_users.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#:reek:TooManyStatements and :reek:UncommunicativeVariableName
+#:reek:FeatureEnvy
+class CreateMusicianSkillUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :musician_skill_users do |t|
+      t.belongs_to :user, index: true
+      t.belongs_to :musician_skills
```
Comment (MariaPaharelava) [inline db/migrate/20190808073147_create_musician_skill_users.rb]: has_many :musician_skill_user ???

File: app/models/musician_skill.rb (hunk 1 above)
Comment (YanPam) [reply app/models/musician_skill.rb]: musician_skill_users work

File: db/migrate/20190808073147_create_musician_skill_users.rb (hunk 2 above)
Comment (als6k) [inline db/migrate/20190808073147_create_musician_skill_users.rb]: t.belongs_to :musician_skill ?

**Comment by longbeardmen** [comment]
This changes are related to PR `DEV004`

**Pull request closed without merging**